.travis.yml

```diff
@@ -39,7 +39,7 @@ matrix:
         directories:
           - dex-contracts/node_modules/
           - $HOME/.npm
-      script: cd dex-contracts && npm ci
+      script: cd dex-contracts && yarn install --frozen-lockfile
```

Comment:
Let's do that switch in a separate PR can we?

Reply:
No, otherwise the build is failing!

Reply:
because the package-lock is no longer available and then npm ci fails

Reply:
When we update to a new `dex-contracts` we also have to update to use `yarn` here.